index.md

```diff
@@ -5,8 +5,8 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
 <div class="hero hero-center hero-main">
     <div class="container hero-wrapper">
         <div class="hero-image animate__animated animate__fadeIn" data-scroll>
-            <img src="/assets/images/home-laptop.svg" alt="Vanilla OS - Showcase" class="on-light" />
-            <img src="/assets/images/home-laptop-dark.svg" alt="Vanilla OS - Showcase" class="on-dark" />
+            <img src="/assets/images/home-laptop.svg" alt="Overview of Vanilla OS with GNOME Maps, GNOME Software and GNOME Weather opened on default style." class="on-light" />
+            <img src="/assets/images/home-laptop-dark.svg" alt="Overview of Vanilla OS with GNOME Maps, GNOME Software and GNOME Weather opened on dark style." class="on-dark" />
```

Comment:
W3C recommends that pure decorative content be ignored by assistive technology. I think this falls under that recommendation.

https://www.w3.org/WAI/tutorials/images/decorative/